Description: Keep building the asm-all artifact to preserve the compatibility with older versions of ASM
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -60,6 +60,10 @@
 // Project descriptions
 // -----------------------------------------------------------------------------
 
+project(':asm-all') {
+  description = parent.description
+}
+
 project(':asm') {
   description = parent.description
   provides = ['org.objectweb.asm', 'org.objectweb.asm.signature']
--- a/settings.gradle
+++ b/settings.gradle
@@ -29,6 +29,7 @@
 rootProject.name = 'root'
 
 include(
+  'asm-all',
   'asm',
   'asm-analysis',
   'asm-commons',
